lib/kino/smart_cell/db_connection.ex

```diff
@@ -0,0 +1,298 @@
+defmodule Kino.SmartCell.DBConnection do
+  @moduledoc false
+
+  # A smart cell used to establish connection to a database.
+
+  use Kino.JS
+  use Kino.JS.Live
+  use Kino.SmartCell, name: "Database connection"
```

Comment:
Should we make it `Kino.JS.SmartCell` for consistency? (though from Livebook perspective it may feel a bit arbitrary)

Reply:
`Kino.SmartCell` is fine IMO. :)